content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -48,10 +57,56 @@ Pipelines use `pipeline-runner` KSA, you can configure IAM permissions of the GS
 
 Pipelines UI uses `ml-pipeline-ui` KSA. If you need to view visualizations stored in Google Cloud Storage (GCS) from pipelines UI, you should add Storage Object Viewer permission to its bound GSA.
 
+#### Cluster setup to use Workload Identity for Pipelines Standalone or GCP Hosted ML pipelines
+
+##### 1. Create your cluster with Workload Identity enabled
+
+* In Google Cloud Console UI, you can enable it in `Create a Kubernetes cluster -> Security -> Enable Workload Identity` like the following:
+<img src="/docs/images/pipelines-auth-gke-enable-workload-identity.png">
+
+* Using `gcloud` CLI, you can enable it with:
+```bash
+gcloud beta container clusters create cluster-name \
+  --release-channel regular \
+  --workload-pool=project-id.svc.id.goog
+```
+
+References:
+
+* [Enable Workload Identity on a new cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster)
+
+* [Enable Workload Identity on an existing cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_an_existing_cluster)
+
+##### 2. Deploy Kubeflow Pipelines
+Deploy via [Pipelines Standalone](/docs/pipelines/installation/overview/#kubeflow-pipelines-standalone) or [GCP hosted ML pipelines](/docs/pipelines/installation/overview#gcp-hosted-ml-pipelines) as usual.
+
+Note, for GCP hosted ML pipelines additionally, you need to patch `proxy-agent` to work with workload identity:
+```
+kubectl patch deployment proxy-agent --patch '{"spec": {"template": {"spec": {"hostNetwork": true}}}}'
```

Comment:
This can be removed once https://github.com/kubeflow/pipelines/pull/3330 is released.

Reply:
same, please remove it for now.

Reply:
I mean just remove it for MKP-UI path (here: "for AI Platform Pipelines additionally").
It's not the designed behavior.

Reply:
removed